src/test/java/com/automationexercise/tests/Tests/AddtocartTest.java

```diff
@@ -0,0 +1,53 @@
+package com.automationexercise.tests.Tests;
+
+import com.automationexercise.DriverManager.DriverFactory;
+import com.automationexercise.DriverManager.DriverManager;
+import com.automationexercise.Pages.CartPage;
+import com.automationexercise.Pages.HomePage;
+import com.automationexercise.Pages.ProductsPage;
+import com.automationexercise.Utilities.DataUtils;
+import com.automationexercise.Utilities.Utility;
+import org.testng.asserts.SoftAssert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class AddtocartTest {
+
+
+    @BeforeMethod
+    public void setup(){
+        DriverFactory.createInstance(DataUtils.getEnvironmentPropertyValue("BROWSER"));
+        Utility.openWebsite(DataUtils.getEnvironmentPropertyValue("BASE_URL"));
+    }
+    @Test
+    public void TC01_addToCart(){
+        new SoftAssert().assertTrue(new HomePage().verifyHomePageVisability());
+        new CartPage().productsNavbarClick();
+        new ProductsPage()
+                .hoverOnBlueTopItemElement()
+                .bluetopddtoCartclick()
+                .continueShoppingButtonClick();
+        new ProductsPage()
+                .hoverOnmenTshirtItemItemElement()
+                .menTshirtAddtocartClick()
+                .continueShoppingButtonClick()
+                .cartNavbarElementClick();
+        new SoftAssert().assertTrue(new CartPage().isBlueTopWomenItemExists());
+        new SoftAssert().assertTrue(new CartPage().isMenShirtItemExists());
+        new SoftAssert().assertTrue(new CartPage().getBlueTopWomenElementIntegerPrice()==500 );
+        new SoftAssert().assertTrue(new CartPage().getMenShirtElementIntegerPrice()==400);
+        new SoftAssert().assertTrue(new CartPage().getBlueTopWomenElementIntegerQuantity()==1);
+        new SoftAssert().assertTrue(new CartPage().getMenShirtElementIntegerQuantity()==1);
+        new SoftAssert().assertTrue(new CartPage().getBlueTopElementIntegerTotalPrice() == new CartPage().getBlueTopWomenElementIntegerQuantity() * new CartPage().getBlueTopWomenElementIntegerPrice() );
```

Comment:
All these operations either comparing the prices or calculating the price should be encapsulated in the PageClass and you just call it here and send the expected result from external JSON file in TestClass here.
Hint: 
```
 public static boolean verifyEquals(By locator, String expectedText) {
        return getText(getDriver(), locator).equals(expectedText);
    }
```
and for calculating price it will be in the PageClass.